charts/tidb-drainer/templates/scripts/_start_drainer.sh.tpl

```diff
@@ -0,0 +1,34 @@
+set -euo pipefail
+
+domain=`echo ${HOSTNAME}`.{{ include "drainer.name" . }}
+
+elapseTime=0
+period=1
+threshold=30
+while true; do
```

Comment:
This part could be an init container. Also it looks like this technically would need the wait-for-pd init container from https://github.com/pingcap/tidb-operator/pull/581 if someone wanted to install this helm chart immediately after the tidb-cluster chart.

Reply:
Agreed, tracked in #749 to keep this PR simple (actually this PR just extracts the drainer statefulset from the tidb-cluster chart)